Wikipedia/Code/StorageAndSyncingSettingsViewController.swift

```diff
@@ -0,0 +1,213 @@
+struct SettingsSection {
+    let headerTitle: String?
+    let footerText: String?
+    let items: [SettingsItem]
+}
+
+struct SettingsItem {
+    let disclosureType: WMFSettingsMenuItemDisclosureType?
```

Comment:
`WMFSettingsMenuItemDisclosureType` has a None case so doesn't need to be optional.

Reply:
I'd rather keep it an optional here

Reply:
Why is that? WMFSettingsMenuItemDisclosureType is an enum. As an optional it gets wrapped inside *another* enum...
> Under the hood optional types are merely an enum with two cases — `None` meaning no value is set and `Some` meaning the value is set and it’s the value associated with it. [sauce](https://hackernoon.com/swift-optionals-explained-simply-e109a4297298)